src/f4enix/output/rssa/rssa_plotting.py

```
@@ -0,0 +1,465 @@
+        )
+        return self
+
+    def get_ratio_to(self, other: "RSSAPlot") -> "RSSAPlot":
+        """Calculate the ratio of the current to another RSSAPlot instance in %
+        difference. Calculated as (other - self) / self * 100. The other RSSAPlot
+        instance should have undergone the same processing as the current one."""
+        if self.raster is None or other.raster is None:
+            raise ValueError(
+                "Raster data is not calculated. Call get_particle_current()"
+                " or other first for both `self` and `other`."
+            )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)